File: cs/Samples/TestDataBuilder/TestUsers.cs
namespace Samples.TestDataBuilder
{
	public class TestUsers
	{
		public static User ARegularUser()
		{
			return new User("Triniti", "tri", "asdasd", "ROLE_USER");
		}

		public static User AnAdmin()
		{
			return new User("Agent Smith", "smith", "qweqwe", "ROLE_ADMIN");
		}
	}
}

File: java/src/test/java/ru/kontur/courses/samples/builder/TestUsers.java
package ru.kontur.courses.samples.builder;

public class TestUsers {

    public static User asRegularUser() {
        return new User("Triniti", "tri", "asdasd", "ROLE_USER");
    }

    public static User anAdmin() {
        return new User("Agent Smith", "smith", "qweqwe", "ROLE_ADMIN");
    }
}
